rasa/core/nlg/callback.py

```diff
@@ -39,11 +39,11 @@ def nlg_request_format(
 ) -> Dict[Text, Any]:
     """Create the json body for the NLG json body for the request."""
     tracker_state = tracker.current_state(EventVerbosity.ALL)
-    response_ids = kwargs.pop(RESPONSE_ID_KEY, [])
+    response_id = kwargs.pop("response_id", "")
 
     return {
         "response": utter_action,
-        "ids": response_ids,
+        "id": response_id,
```

Comment:
Good point, I made a decision for an empty string because I thought that I can't send a None value in a POST request body, but maybe I got confused by some other testing issues encountered while I was working on the implementation. I will double-check now if None is possible and also ask CSE if they prefer one or the other.

Reply:
Sounds good!

Reply:
None seems to be possible, so I've pinged CSE on the ticket to enquire which option they prefer. In the meantime, I'll re-request review 😄